app/models/concerns/foreman_azure_rm/vm_extensions/managed_vm.rb

```diff
@@ -1,7 +1,7 @@
 # This concern has the methods to be called inside azure_rm.rb
 module ForemanAzureRm
-  module VMExtensions
-    module ManagedVM
+  module VmExtensions
+    module ManagedVm
```

Comment:
Makes me question if VM should also be an acronym we recognize

Reply:
@ofedoren what do you think? I suppose we should be consistent across all Foreman, hence adding the acronym to the main repo.

Reply:
Done: https://github.com/theforeman/foreman/pull/10131/commits/c17812e186ccb244d739aaca67d80453f6072826